cypress/integration/tests/developer/applicationinventory/assessment/filter.test.ts

```diff
@@ -182,13 +186,6 @@ describe("Application inventory filter validations", { tags: "@tier2" }, functio
         clickByText(button, clearAllFilters);
 
         // Enter a non-existing tag and apply it as search filter
-        applySearchFilter(tag, data.getRandomWord(5));
-        cy.wait(3000);
-
-        // Assert that no search results are found
-        cy.get("h2").contains("No applications available");
-
-        // Clear all filters
```

Comment:
"applySelectFilter" method contains  a non-existing tag and apply it as search filter, So removed applySearchFilter and reduced the code

Reply:
Why are we removing this validation ? Invalid tags can be searched.

Reply:
Validation is also include or present in applySelectFilter method  So, we don't need this extra lines now  ```
inputText("input.pf-c-form-control.pf-m-search", filterText);
    if (isValid) {
        clickByText("span.pf-c-check__label", filterText);
    } else {
        cy.contains("div.pf-c-select__menu", "**No results found**");
``` 
![Screenshot from 2022-09-13 08-04-22](https://user-images.githubusercontent.com/87791967/189795010-edac28ea-56d1-4736-88b1-d6e2fb9378bc.png)